src/components/Link.js

```diff
@@ -26,15 +26,27 @@ class CSVLink extends React.Component {
    * In IE11 this method will trigger the file download
    */
   handleLegacy(evt, data, headers, separator, filename) {
+    // Always use blob to handle large chunks of data
+    let blob = new Blob([toCSV(data, headers, separator)])
     // If this browser is IE 11, it does not support the `download` attribute
     if (window.navigator.msSaveOrOpenBlob) {
       // Stop the click propagation
       evt.preventDefault()
-
-      let blob = new Blob([toCSV(data, headers, separator)])
       window.navigator.msSaveBlob(blob, filename)
 
       return false
+    } else {
```

Comment:
`handleLegacy` is responsible for handling cases where the anchor element's `href` attribute cannot be used to trigger the download. This `else` block is unnecessary as we could just move this logic to the `buildURI` function and have it return that straight to the `href` attribute on line 58. That would reduce the complexity of the changes from this PR which would subsequently make it easier for the code coverage to remain above 90%. 

Let me know if you are able to make those changes, otherwise I am happy to implement them myself in a new PR (as I have been meaning to do)

Reply:
Agreed.  Arguably the function should be renamed, since the function's purpose is to support IE11, which requires us to use msSaveBlob, as it doesn't support href="data:"